README.md

```diff
@@ -62,6 +62,10 @@ There are two main categories of samples: **Camera** and **Applications**. The s
     - [**CreateDepthMap**][CreateDepthMap-url] - Convert point cloud from ZDF file to OpenCV format, extract depth map and visualize it.
       - **Dependencies:**
         - [OpenCV](https://opencv.org/) version 4.1.0 or newer
+    - [**MaskPointCloud**][MaskPointCloud-url] - Mask point cloud from ZDF file and convert to PCL format, extract depth map and visualize it.
```

Comment:
Mask point cloud from ZDF file, convert to PCL format, extract depth map and visualize it.

Reply:
The reason I wrote like this is because the convertion is during the masking. But maybe ok to write like yours either way?

Reply:
I see your point. I don't think it matters. It is consistency vs precision. Your call.